react/fields/MonthPicker/MonthPicker.js

```diff
@@ -0,0 +1,175 @@
+import styles from './MonthPicker.less';
+
+import React, { Component, PropTypes } from 'react';
+import classnames from 'classnames';
+
+import ErrorIcon from '../../icons/ErrorIcon/ErrorIcon';
+import CustomMonthPicker from './CustomMonthPicker/CustomMonthPicker';
+import NativeMonthPicker from './NativeMonthPicker/NativeMonthPicker';
+
+function combineClassNames(props = {}, ...classNames) {
+  const { className, ...restProps } = props;
+
+  return {
+    className: classnames.apply(null, [...classNames, className]), // eslint-disable-line no-useless-call
+    ...restProps
+  };
+}
+
+export default class MonthPicker extends Component {
```

Comment:
Having looked at this component I wonder whether it needs to be a `composite`. It seems to just be a proxy for the `custom` and the `native` component. What are your thoughts? It could be a functional component? Maybe I'm missing something but it looks like it only renders.